ui/explore/src/components/ExploreManager/ExploreManager.tsx

```diff
@@ -0,0 +1,48 @@
+// Copyright 2023 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { Stack, Tab, Tabs } from '@mui/material';
+import { ReactNode, useState } from 'react';
+import { ExploreToolbar } from '../ExploreToolbar';
+import { TracesExplorer } from './TracesExplorer';
+import { TimeSeriesExplorer } from './TimeSeriesExplorer';
+
+export interface ExploreManagerProps {
+  exploreTitleComponent?: ReactNode;
+}
+
+export function ExploreManager(props: ExploreManagerProps) {
+  const { exploreTitleComponent } = props;
+  const [tabState, setTabState] = useState(0);
+
+  return (
+    <Stack sx={{ width: '100%' }} px={2} pb={2} pt={1.5} gap={3}>
+      <ExploreToolbar exploreTitleComponent={exploreTitleComponent} />
+
+      <Stack direction="row" gap={2}>
+        <Tabs
+          orientation="vertical"
```

Comment:
Not a fan of the vertical tabs here, we're losing space on the left & to display graphs it's always a good thing to be able to take more space 😄

Reply:
In my opinion it lets a large space for the graph. I added a media adapter to be responsive in case we are in a mobile, but I don't know what I can do more. 

Maybe if we could reduce the tabs with a << that would change them to pictograms?

Reply:
Well I was thinking to go for horizontal tabs in all the cases. I also feel this would be better for overall UX consistency, since we went for this design for the admin & project pages. But having this kind of switch based on screen size is also fine for me.
Btw I'm just thinking about it now, what if you want to investigate both metrics & traces at the same time? Does it keep the other-tab's context when you are switching tabs?

Reply:
Funny that you talk about using the explorer for traces and metrics at the same time as it's been a reflex by Guillaume as well. 

To answer directly to your question, the context is lost if you switch from one tab to another, but it could be interesting to prevent that like saving the data in memory. We do this kind of thing in the query selection to not loose the previous query. I will think about it and make another PR for that.

Reply:
As discussed, updated with a card to highlight better the main content. Screenshots has been updated. 
I'm also adding to my todo list to save the last queries before switching tab.